migrations/2_deploy_contracts.js

```diff
@@ -1,15 +1,14 @@
-const FanDonation = artifacts.require("./FanDonation.sol");
-const FanProxy = artifacts.require("./FanProxy.sol");
-const FanSwap = artifacts.require("./FanSwap.sol");
+const CreatorCashout = artifacts.require("./CreatorCashout.sol");
 
 // 0x88757f2f99175387ab4c6a4b3067c77a695b0349 is the address for LendingPoolAddressesProvider
 // on Kovan. (https://docs.aave.com/developers/deployed-contracts)
 const LendingPoolAddressesProvider =
   "0x88757f2f99175387ab4c6a4b3067c77a695b0349";
 
 module.exports = function (deployer) {
-  deployer.deploy(FanDonation, LendingPoolAddressesProvider);
-  deployer.deploy(FanSwap).then(() => {
-    return deployer.deploy(FanProxy, FanSwap.address, FanDonation.address);
-  });
+  deployer.deploy(CreatorCashout, LendingPoolAddressesProvider);
+  //deployer.deploy(FanSwap).then(() => {
```

Comment:
You don't need to comment these out here, just make sure they're commented out in your local branch when you're doing `truffle migrate --network kovan --reset`

Reply:
Honestly i shoulda not included truffle changes in this commit